substreams/ethereum-uniswap-v4/shared/src/modules/6_map_protocol_changes.rs

```diff
@@ -155,16 +153,14 @@ pub fn map_protocol_changes(
             });
         });
 
-    let block_storage_changes = get_block_storage_changes(&block);
-
     Ok(BlockChanges {
         block: Some((&block).into()),
         changes: transaction_changes
             .drain()
             .sorted_unstable_by_key(|(index, _)| *index)
             .filter_map(|(_, builder)| builder.build())
             .collect::<Vec<_>>(),
-        storage_changes: block_storage_changes,
+        storage_changes: vec![],
```

Comment:
I'd add a dev note here ntoing that this is required for hooks and will be enriched in a further module